doc/nrf/ug_nfc.rst

```diff
@@ -0,0 +1,222 @@
+   Similarly, the ME flag is set for the last record in the message.
+   If a record is the only record in a message, both flags are set.
+CF (Chunk Flag):
+   Used for chunked payloads (a payload that is partitioned into multiple records).
+   Set in all chunks of the record except for the last one.
+   Note, however, that chunking is not supported by this library.
+SR (Short Record):
+   Used to determine the size of the payload length field.
+   If the flag is set, the Payload Length occupies 1 byte; otherwise it is 4 bytes long.
+   Note that the NDEF generator supports a Payload Length of 4 bytes only at the moment.
+IL (ID Length present):
+   Indicates whether an ID Length field is present in the header.
+   If the flag is set, the ID Length field is present.
+TNF (Type Name Format):
+   Specifies the structure of the Payload Type field and how to interpret it.
+   The following values are allowed (square brackets contain documentation reference related to the specific type):
+
+   +-------+------------------------------------+
+   | Value | Type Name Format                   |
+   +=======+====================================+
+   | 0x00  | Empty                              |
+   +-------+------------------------------------+
+   | 0x01  | NFC Forum well-known type [NFC RTD]|
+   +-------+------------------------------------+
+   | 0x02  | Media-type [RFC 2046]              |
+   +-------+------------------------------------+
+   | 0x03  | Absolute URI [RFC 3986]            |
+   +-------+------------------------------------+
+   | 0x04  | NFC Forum external type [NFC RTD]  |
+   +-------+------------------------------------+
+   | 0x05  | Unknown                            |
+   +-------+------------------------------------+
+   | 0x06  | Unchanged                          |
+   +-------+------------------------------------+
+   | 0x07  | Reserved                           |
+   +-------+------------------------------------+
+
+.. _ug_nfc_ndef_format_payload:
+
+Record payload
+==============
+
+The content of the payload is application-specific and related to the type of the record.
+For example, in URI records, the payload contains a web address of the page that the polling device should open.
+
+Note that the payload of an NDEF record can contain a nested NDEF message.
+This nested message must be a full NDEF message, consisting of one or multiple NDEF records with the appropriate setting of MB and ME flags.
+
+
+.. _ug_nfc_tnep:
+
+Tag NDEF Exchange Protocol (TNEP)
+*********************************
+
+The Tag NDEF Exchange Protocol (TNEP) is an application-level protocol for sending or retrieving application data units between a polling (reader) device and an NFC tag.
+The data units are NFC Data Exchange Format (NDEF) messages.
+
+The protocol operates between the NDEF application layer and the tag.
+The tag must be a device that supports at least one communication protocol for NDEF.
```

Comment:
if we want to be hypercorrect then all Tag types work except for Type 1 Tag

Reply:
that supports NDEF data exchange
->
that supports NDEF Read and Write Procedure

Bidirectional NDEF data exchange is provided by TNEP so this might sound confusing